README.md

```diff
@@ -0,0 +1,3 @@
+# wikiparser
+
+_A parser to extract articles from Wikipedia database dumps._
```

Comment:
... for later embedding into mwm map files created by Organic Maps generator (link here)

Reply:
How's this?
```suggestion
_Extracts articles from [Wikipedia database dumps](https://en.wikipedia.org/wiki/Wikipedia:Database_download) for embedding into the `mwm` map files created by [the Organic Maps generator](https://github.com/organicmaps/organicmaps/blob/master/tools/python/maps_generator/README.md)._
```